devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -73,6 +73,7 @@ metadata {
 		fingerprint mfr:"037B", prod:"0003", model:"0001", deviceJoinName: "KeyWe Door Lock" // GKW-1000Z //KeyWe Smart Rim Lock
 		// Philia
 		fingerprint mfr:"0366", prod:"0001", model:"0001", deviceJoinName: "Philia Door Lock" // PDS-100 //Philia Smart Door Lock
+        fingerprint mfr: "0123", prod: "0001", model: "2222", deviceJoinName: "Door Lock"
```

Comment:
**Please:**
- fix spacing (change 4 spaces -> 1 tab),
- change deviceJoinName to "U-Tec Door Lock" 
- add brand name and model name in a comment after Your fingerprint
- add the Raw Description (it can be found at graph.api.smartthings.com-> my devices -> ) in a comment after Your fingerprint -> // U-Tec UBZ01, Raw Description ...